### PR TITLE
[fix] Correct the content and Update the angle bracket linking in glossary page and correct link formatting

### LINKED ISSUE #83: Content swapped or in wrong places
I found that in the installation documentation, instead of having the Windows installation docs (https://www.jenkins.io/doc/book/installing/windows/) it contains the Windows support policy:

Screenshot 2023-12-15 at 5 18 35 PM

Additionally, the Blue Ocean "Getting started with Blue Ocean" link is now "Getting started with the guided tour" which is a completely separate part of documentation:

Screenshot 2023-12-15 at 2 06 40 PM
Screenshot 2023-12-15 at 2 06 51 PM

It also looks like none of the content above the "Properties in Jenkins core" is present in the versioned docs site:

Screenshot 2023-12-15 at 2 27 47 PM
Screenshot 2023-12-15 at 2 27 53 PM

Finally, in the Defining a Pipeline section of the "Getting started with Pipeline" page, the link formatting of the pipeline creation options is changed to have brackets instead of normal formatting:

Screenshot 2023-12-15 at 10 56 49 AM
Screenshot 2023-12-15 at 10 57 20 AM

### PR DESCRIPTION
**Pull request by Vandit1604**
Fixes #83 and  #80  
with Removed usage of angle brackets in glossary and some pages

### CODE REVIEW

**Comment by Vandit1604** [comment]
I'm using this PR for both the issues. The effort is to remove the usage of angle brackets linking as they make the links navigate to top of the page if the resource id isn't found and doesn't raise any error in antora logs.

**Comment by krisstern** [comment]
> I'm using this PR for both the issues. The effort is to remove the usage of angle brackets linking as they make the links navigate to top of the page if the resource id isn't found and doesn't raise any error in Antora logs.

I don't understand the context: why only part of #80 is resolved while you claim here both issues?

File: docs/projects/modules/gsoc/pages/2021/project-ideas/remoting-monitoring.adoc [hunk 1]
```diff
@@ -1,17 +1,34 @@
...
+This project idea is published as a Google doc. You are welcome to comment on the proposal or to suggest changes in the draft embedded below ( https://docs.google.com/document/d/1nhb-lfdjxT2fAY3US15HIMvePQCCoWU82VV1Y8Mu3lc[Google Doc] )
 
+++++
+<iframe src="https://docs.google.com/document/d/1nhb-lfdjxT2fAY3US15HIMvePQCCoWU82VV1Y8Mu3lc" width="100%" height="600px"></iframe>
+++++
 
-# To be confirmed
-#- "linuxsuren"
-#- "mikecirioli"
-#- "markyjackson-taulia"
```
Comment (krisstern) [inline docs/projects/modules/gsoc/pages/2021/project-ideas/remoting-monitoring.adoc:13]: Why are these deleted?

**Review by krisstern — CHANGES_REQUESTED** [review]

File: docs/projects/modules/gsoc/pages/2021/project-ideas/remoting-monitoring.adoc (hunk 1 above)
Comment (Vandit1604) [reply docs/projects/modules/gsoc/pages/2021/project-ideas/remoting-monitoring.adoc:13]: These are comments. If we want them. I can add them back.

**Comment by Vandit1604** [comment]
> I don't understand the context: why only part of #80 is resolved while you claim here both issues?

When I raised this PR, I fixed the angle brackets links that were in glossary page which are in this [issue](https://github.com/Vandit1604/jenkins-docs/issues/80#issue-2044430868) submitted by kevin.

With this commit I have fixed the rest of link navigation.
Also I have commented some things on that issue related to some links 
See here https://github.com/Vandit1604/jenkins-docs/issues/80#issuecomment-1858939768

File: docs/projects/modules/gsoc/pages/2021/project-ideas/remoting-monitoring.adoc (hunk 1 above)
Comment (krisstern) [reply docs/projects/modules/gsoc/pages/2021/project-ideas/remoting-monitoring.adoc:13]: These are not just comments, instead they are comments of historical significance. Please add these back.

**Comment by krisstern** [comment]
> > I don't understand the context: why only part of #80 is resolved while you claim here both issues?
> 
> When I raised this PR, I fixed the angle brackets links that were in glossary page which are in this [issue](https://github.com/Vandit1604/jenkins-docs/issues/80#issue-2044430868) submitted by kevin.
> 
> With this commit I have fixed the rest of link navigation. Also I have commented some things on that issue related to some angle links See here [#80 (comment)](https://github.com/Vandit1604/jenkins-docs/issues/80#issuecomment-1858939768)

Now I see the logic! Thanks for clarifying @Vandit1604

**Review by krisstern — APPROVED** [review]
